src/main/java/com/flair/bi/service/SchedulerService.java

```diff
@@ -82,7 +85,22 @@
 			JSONObject jsonObject = new JSONObject();
 			return ResponseEntity.status(responseEntity.getStatusCode()).body(responseEntity.getBody());
 		}
-	}	
+	}
+	
+	public ResponseEntity<String> searchScheduledReport(String userName,String reportName,String startDate,String endDate,Integer pageSize, Integer page) {
+		RestTemplate restTemplate = new RestTemplate();
+		ResponseEntity<String> responseEntity = null;
+		SchedulerResponse schedulerResponse= new SchedulerResponse();
+		try {
+			responseEntity = restTemplate.exchange(buildUrl(host, port, searchscheduledReportsURL), HttpMethod.GET, null,
+					String.class, userName,reportName,startDate, endDate,pageSize, page);
+			return ResponseEntity.status(responseEntity.getStatusCode()).body(responseEntity.getBody());
+		} catch (Exception e) {
+			log.error("error occured while getting scheduled report logs:"+e.getMessage());
+			JSONObject jsonObject = new JSONObject();
+			return ResponseEntity.status(responseEntity.getStatusCode()).body(responseEntity.getBody());
```

Comment:
are sending error thrown by notification to UI...can u check this?

Reply:
yes, the error message thrown by notification will be show to UI